crates/phactory/src/prpc_service.rs

```diff
@@ -886,8 +891,14 @@ impl<Platform: pal::Platform + Serialize + DeserializeOwned> Phactory<Platform>
             info: Some(pb::ClusterInfo {
                 id: hex(cluster.id),
                 state_root: cluster.storage.root().map(hex).unwrap_or_default(),
-                contracts: contracts.keys().map(hex).collect(),
```

Comment:
Should we retain this field until the commit is deployed to mainnet? I'm worried that removing it might lead to a protobuf schema update, potentially causing compatibility issues when updating the SDK for this purpose.